src/Hash.h

```diff
@@ -170,6 +170,11 @@ Data groestl512(const T& data) {
     return groestl512(reinterpret_cast<const byte*>(data.data()), data.size());
 }
 
+/// Computes the SHA256 hash.
+inline Data sha256s(const byte* data, size_t size) {
```

Comment:
After some experimenting, I _believe_ it's to help out the compiler over in `CB58.h` where a `Hasher` is supplied, otherwise compilation fails. (@kylecbetts, do I have that right?) Maybe there's a better way to supply the correct `sha256` over in `CB58.h`.

Reply:
@catenocrypt @EvanJRichard I believe my thought process was to use the **inline** keyword to potentially improve the performance of the function call like the other hash functions are currently using. Ultimately, a hasher needs passed into our CB58 class with the default currently being the **inline** version of sha256. This does not need to be the inline version, it could simply be the original sha256 function if we feel that is more appropriate.

Reply:
Modern compilers 'guess' the inline, the inline keyword is just like a hint.  I think for other similar functions the 'duplicated'version is needed for template speciation, but in this case they both happen to have the same signature.  But it can stay, it's a minor issue.